### DISCUSSION TITLE
Direct link to order pcb

### DISCUSSION THREAD

**Discussion by kloel** [AxxAxx/AxxSolder#41] [Ideas]
Is there a link direct to any pcb manufacturer?

Easyer to order the pcb then downloading the pcb layout and upload it to the manufacturer

**Comment by AxxAxx** [comment]
Now there is no link, I could ask PCBWay if they can have a standing order or something. 

For now you have all needed gerber files in a zip archive here: https://github.com/AxxAxx/AxxSolder/tree/main/AxxSolder_hardware/fabrication

Just upload that zip archive to any PCB manufacturer. A suggestion is to add a stencil too.
